x-pack/plugins/apm/public/components/app/Correlations/LatencyCorrelations.tsx

```diff
@@ -0,0 +1,271 @@
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="s">
+                <h4>Latency distribution</h4>
+              </EuiTitle>
+              <LatencyDistributionChart
+                data={data}
+                status={status}
+                selectedSignificantTerm={selectedSignificantTerm}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <SignificantTermsTable
+            significantTerms={data?.significantTerms}
+            status={status}
+            setSelectedSignificantTerm={setSelectedSignificantTerm}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </>
+  );
+}
+
+function LatencyTimeseriesChart({
+  data,
+  selectedSignificantTerm,
+  status,
+}: {
+  data?: CorrelationsApiResponse;
+  selectedSignificantTerm: SignificantTerm | null;
+  status: FETCH_STATUS;
+}) {
+  if (!data) {
+    if (status === FETCH_STATUS.LOADING) {
+      return <EuiLoadingChart size="m" />;
+    }
+
+    return <div>no data for chart</div>;
+  }
+
+  const dateFormatter = timeFormatter('HH:mm:ss');
+
+  const yValues = [
+    ...data.overall.timeseries.map((p) => p.y ?? 0),
+    ...data.significantTerms.flatMap((term) =>
+      term.timeseries.map((p) => p.y ?? 0)
+    ),
+  ];
+  const yMax = Math.max(...yValues);
+  const durationFormatter = getDurationFormatter(yMax);
+
+  return (
+    <Chart size={{ height: px(200), width: px(600) }}>
+      <Settings showLegend legendPosition={Position.Bottom} />
+
+      <Axis
+        id="bottom"
+        position={Position.Bottom}
+        showOverlappingTicks
+        tickFormat={dateFormatter}
+      />
+      <Axis
+        id="left"
+        position={Position.Left}
+        domain={{ min: 0, max: yMax }}
+        tickFormat={(d) => durationFormatter(d).formatted}
+      />
+
+      <LineSeries
+        id="Overall latency"
+        xScaleType={ScaleType.Time}
+        yScaleType={ScaleType.Linear}
+        xAccessor={'x'}
+        yAccessors={['y']}
+        data={data.overall.timeseries}
+        curve={CurveType.CURVE_MONOTONE_X}
+      />
+
+      {selectedSignificantTerm !== null ? (
+        <LineSeries
+          id="Latency for selected term"
+          xScaleType={ScaleType.Time}
+          yScaleType={ScaleType.Linear}
+          xAccessor={'x'}
+          yAccessors={['y']}
+          color="red"
+          data={selectedSignificantTerm.timeseries}
+          curve={CurveType.CURVE_MONOTONE_X}
+        />
+      ) : null}
+    </Chart>
+  );
+}
+
+function LatencyDistributionChart({
+  data,
+  selectedSignificantTerm,
+  status,
+}: {
+  data?: CorrelationsApiResponse;
+  selectedSignificantTerm: SignificantTerm | null;
+  status: FETCH_STATUS;
+}) {
+  if (!data) {
+    if (status === FETCH_STATUS.LOADING) {
+      return <EuiLoadingChart size="m" />;
+    }
+
+    return <div>no data for chart</div>;
```

Comment:
See above about the container. Elastic charts has a default empty state which we should use instead of making our own (it shows up when the data are empty.)